module/planning/PlanWalkPath/data/config/webots/PlanWalkPath.yaml

```diff
@@ -1,9 +1,9 @@
 # Controls the minimum log level that NUClear log will display
-log_level: DEBUG
+log_level: INFO
 
 # General playing tuning values for approaching ball
 max_forward_speed: 0.5
-min_forward_speed: 0.1
+min_forward_speed: 0.2
```

Comment:
were you able to get it to stop falling over on start

Reply:
I can start mine in initial for a bit prior to going into ready, we should add some behaviour to only start walking when arms are down.